assets/js/common.js

```diff
@@ -201,6 +201,7 @@ var hivepress = {
 					if (parentField.length) {
 						parentField.on('change', function () {
 							field.data('parent-value', $(this).val());
+							field.val(null).trigger('change');
```

Comment:
Please explain what was the reason of the issue and how this code fixes it in detail

Reply:
As u can see this code is clearing select2 field value